git-php.php

```diff
@@ -363,7 +363,7 @@ class="max" value="<?php if (isset($_POST['email'])) echo clean($_POST['email'])
 $purposes = ["Learning PHP", "Coding in PHP", "Reading the PHP source",
     "Using PHP extensions", "Creating experimental PHP extensions",
     "Submitting a patch to PHP", "Adding notes to the documentation",
-    "Writing web pages with PHP"];
+    "Writing web pages with PHP", ];
```

Comment:
@localheinz is there a fixer or a PR already for:
```suggestion
    "Writing web pages with PHP",
];
```
?